src/client/components/ProductsList/actions.js

```diff
@@ -0,0 +1,12 @@
+import { ENDPOINT_URL } from '../../common/variables';
+
+// Action types
+export const FETCH_ITEMS = 'FETCH_ITEMS';
+
+// Action creators
+export const recieveItems = json => ({ type: FETCH_ITEMS, items: json });
+
+export const fetchItems = () => dispatch =>
+  fetch(`${ENDPOINT_URL}/items`, { credentials: 'same-origin' })
+    .then(resp => resp.json())
+    .then(json => dispatch(recieveItems(json)));
```

Comment:
what about error handling in catch?

Reply:
temporarily it will be just console.error, later on the next branch I'll dispatch action FETCH_FAILED and set suitable status to false